test/onnx/test_onnx_opset.py

```diff
@@ -369,6 +370,31 @@ def forward(self, x):
         x = torch.randn(20, 16, 50)
         check_onnx_opsets_operator(MyDynamicModel(), x, ops, opset_versions=[9, 10])
 
+    def test_grid_sample(self):
+        n, c, h_in, w_in, h_out, w_out = 1, 1, 3, 2, 2, 4
+        ops = [{"op_name": "GridSample"}]
+        ops = {16: ops}
+
+        class MyModule(Module):
```

Comment:
Possible to use a more descriptive name for the module? E.g. `GridSampleModule`, `TestGridSampleModule` etc.